lib/kcl.js

```diff
@@ -231,7 +230,8 @@ module.exports = function(config, kinesis) {
         if (!resp.data.NextShardIterator && resp.data.Records) {
           db.shardComplete(shard, function (err) {
             if (err) throw err;
-            console.log('Shard', shard.id, 'marked as complete');
+            shard.status = 'complete';
```

Comment:
Sure, I've added a small change in the latest commit to address this. It follows the same pattern `init` and `processRecords` (with a callback).